tests/python/test_ad_grad_check.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import pytest
+
+import taichi as ti
+from tests import test_utils
+
+
+@test_utils.test()
+def test_grad_check():
```

Comment:
btw ia nice followup is that we add tests for major ops (which support AD) so that we make sure no regression is happening.